Makefile

```
@@ -251,6 +252,10 @@ $(RELEASE_NOTES): ## Build release notes.
 $(GO_APIDIFF): ## Build go-apidiff from tools folder.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) github.com/joelanford/go-apidiff $(GO_APIDIFF_BIN) $(GO_APIDIFF_VER)
 
+conversion-verifier: $(CONVERSION_VERIFIER) go.mod go.sum ## fetch CAPI's conversion verifier
+$(CONVERSION_VERIFIER): go.mod
```

```suggestion
$(CONVERSION_VERIFIER): $(TOOLS_DIR)/go.mod ## fetch CAPI's conversion verifier
```
I think we don't need / shouldn't add the conversion-verifier target